taxi/views.py

```diff
@@ -64,7 +83,7 @@ class CarDetailView(LoginRequiredMixin, generic.DetailView):
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarsCreatedForm
```

Comment:
The form class `CarsCreatedForm` used here should be renamed to `CarCreationForm` to maintain consistency with naming conventions and the rest of the codebase.